src/object_properties.cpp

```diff
@@ -71,6 +73,9 @@ void ObjectProperties::serialize(std::ostream &os) const
 	writeF1000(os, weight);
 	writeV3F1000(os, collisionbox.MinEdge);
 	writeV3F1000(os, collisionbox.MaxEdge);
+	writeV3F1000(os, selectionbox.MinEdge);
+	writeV3F1000(os, selectionbox.MaxEdge);
+	writeU8(os, pointable);
```

Comment:
This is only possible by adding the new values in this output-stream below the already existing ones. Otherwise, older clients will read garbage from the new "gap" you opened here.